### PR TITLE
Dockerize

### PR DESCRIPTION
**Pull request by cavis**
Upgrades!  Docker!  Everything!

### CODE REVIEW

File: bin/get-ffmpeg [hunk 1]
```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
-FFMPEG_BIN="https://johnvansickle.com/ffmpeg/releases/ffmpeg-release-64bit-static.tar.xz"
+FFMPEG_BIN="https://s3.amazonaws.com/prx-tech/archives/ffmpeg-release-64bit-static.tar.xz"
```
Comment (kookster) [inline bin/get-ffmpeg]: this is such a good idea

File: bin/get-ffmpeg [hunk 2]
```diff
@@ -14,8 +14,8 @@ if [ ! -f bin/ffprobe ]; then
   mkdir -p bin
   mv ffmpeg-*-static/ffprobe bin/
   rm -rf ffmpeg-*-static
-  if [ ! -f bin/ffprobe ]; then
-    echo "ERROR: something went wrong - ffprobe not present"
+  if [[ ! -f bin/ffprobe ]]; then
+    echo "ERROR: something went wrong - ffmmpeg not present"
```
Comment (kookster) [inline bin/get-ffmpeg]: checking for ffprobe, but error is ffmpeg?

Comment (kookster) [reply bin/get-ffmpeg]: (and `ffmpeg` is misspelled)

Comment (cavis) [reply bin/get-ffmpeg]: Changed the message to `ffprobe`.  This script is pretty much copied from the dovetail-stitch-lambda, and i didn't want to change it too much yet.  (And this repo will likely need `ffmpeg` too in the near future).

**Pull request merged**